src/main/java/org/tikv/common/TiSession.java

```diff
@@ -159,23 +159,46 @@ private static VersionInfo getVersionInfo() {
 
   private synchronized void warmUp() {
     long warmUpStartTime = System.currentTimeMillis();
+    warmUpStore();
```

Comment:
would it be better to handle exceptions in this function since all the exceptions are ignored? for example:

```java
try {
  warmUpStore();
  warmUpRegion();
  ...
} catch () {
  ...
}
```

Reply:
if `warmUpStore` failes, `warmUpRegion` will still run.

Reply:
ok